dgraph/graphql/src/dgraph/queries.js

```diff
@@ -0,0 +1,19 @@
+export const queries = {
+  entities: (filter) => {
+    const type = filter?.type ?? "medicinal_product";
+    const codeFilter = filter?.code ? `@filter(eq(code, "${filter.code}"))` : ``;
+
+    return `{
+      query(func: eq(type, ${type})) ${codeFilter} @recurse(loop: false)  {
```

Comment:
I just realised this won't work if we want to search by code, but don't know what level in the tree we're looking at. I'm thinking the filtering we want here is dependent on what we want to do in the UI (or the UI can also request everything and apply it's own filtering...but could become quite large).